File: Kata 6/Title Case/Title Case.cs
using System.Linq;
public class Kata
{
    public static string TitleCase(string title, string minorWords = "")
    {

        if (title == string.Empty)
            return "";
        //*****¥i¥H¨¾¤î minorWords==null
        var minWords = (minorWords ?? "").Split(' ').Select(w => w.ToLower());
        var q = title.ToLower().Split(' ').Select((x, i) => (minWords.Contains(x) && i != 0) ? x: x.Substring(0, 1).ToUpper() + x.Substring(1));
        return string.Join(" ", q);
    }	
}

File: Kata 6/Title Case/Title Case.py
def title_case(title, minor_words=""):
    return " ".join( s.title() if s  not in minor_words.lower().split(' ') or i==0 else s for i,s in enumerate(title.lower().split(' ') ))
    
def title_case(title, minor_words=''):
    words=[ word.lower() for word in minor_words.split(' ')]
    titles=title.split(' ')
    for i,t in enumerate(titles):
        if i!=0 and t.lower()  in words:
            titles[i]=titles[i].lower()
        else:
            titles[i]=titles[i].title()
    return ' '.join(titles)